x-pack/plugins/observability/public/components/app/section/index.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiAccordion, EuiLink, EuiPanel, EuiSpacer, EuiText, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import styled from 'styled-components';
+import { ErrorPanel } from './error_panel';
+
+interface Props {
+  title: string;
+  subtitle: string;
+  hasError: boolean;
+  children: React.ReactNode;
+  minHeight?: number;
+  appLink?: string;
+}
+const StyledEuiAccordion = styled(EuiAccordion)`
+  .euiAccordion__triggerWrapper {
+    border-bottom: ${(props) => props.theme.eui.euiBorderThin};
+  }
+  .euiAccordion__button {
+    margin-bottom: 16px;
+  }
+  .euiAccordion__childWrapper {
+    margin-top: 8px;
+  }
+`;
+
+export const SectionContainer = ({
+  title,
+  appLink,
+  children,
+  subtitle,
+  minHeight,
+  hasError,
+}: Props) => {
+  return (
+    <StyledEuiAccordion
```

Comment:
When sections are collapsed I don't think they should take up this much space

![image](https://user-images.githubusercontent.com/209966/86615322-98fda680-bfb4-11ea-8676-61b80bec7967.png)

Reply:
Already fixed. Thanks for the heads up